tensorflow/lite/delegates/external/external_delegate.cc

```diff
@@ -155,12 +155,12 @@ ExternalDelegateWrapper::ExternalDelegateWrapper(
                                               ckeys.size(), nullptr);
     if (external_delegate_) {
       wrapper_delegate_ = {
-          .data_ = reinterpret_cast<void*>(this),
-          .Prepare = DelegatePrepare,
-          .CopyFromBufferHandle = nullptr,
-          .CopyToBufferHandle = nullptr,
-          .FreeBufferHandle = nullptr,
-          .flags = external_delegate_->flags,
+          reinterpret_cast<void*>(this),    // .data =
+          DelegatePrepare,                  // .Prepare =
+          nullptr,                          // .CopyFromBufferHandle =
+          nullptr,                          // .CopyToBufferHandle =
+          nullptr,                          // .FreeBufferHandle =
+          external_delegate_->flags,        // .flags =
```

Comment:
As designated initializers are only supported on MSVC 2019 with stdc++20 and this is the only use of the syntax in the codebase, I modified to use the standard syntax and kept the field names as comments for readability.